addons/arsenal/functions/fnc_addVirtualItems.sqf

```diff
@@ -81,37 +81,37 @@ if (_items isEqualType true) then {
                         /* Weapon acc */
                         case (
                                 isClass (_configItemInfo) &&
-                                {(getNumber (_configItemInfo >> "type")) in [101, 201, 301, 302]} &&
+                                {(getNumber (_configItemInfo >> "type")) in [TYPE_MUZZLE, TYPE_OPTICS, TYPE_FLASHLIGHT, TYPE_BIPOD]} &&
                                 {!(_x isKindOf ["CBA_MiscItem", (_configCfgWeapons)])}
                             ): {
                             switch (getNumber (_configItemInfo >> "type")) do {
-                                case 201: {
+                                case TYPE_OPTICS: {
                                     (_cargo select 1) select 0 pushBackUnique _x;
                                 };
-                                case 301: {
+                                case TYPE_FLASHLIGHT: {
                                     (_cargo select 1) select 1 pushBackUnique _x;
                                 };
-                                case 101: {
+                                case TYPE_MUZZLE: {
                                     (_cargo select 1) select 2 pushBackUnique _x;
                                 };
-                                case 302: {
+                                case TYPE_BIPOD: {
                                     (_cargo select 1) select 3 pushBackUnique _x;
                                 };
                             };
                         };
                         /* Headgear */
                         case (isClass (_configItemInfo) &&
-                            {getNumber (_configItemInfo >> "type") == 605}): {
+                            {getNumber (_configItemInfo >> "type") == TYPE_HEADGEAR}): {
```

Comment:
isEqualTo

Reply:
these functions can't gain noticeable performance (nanoseconds in comparison to overall microseconds) from this replacement but become really less readable